Automattic/wp-calypso
# Stats: Move data requests etc to the real UTM module

Related: p1HpG7-rOR-p2#3-utm-takes-too-long-to-load

## Proposed Changes

* Move most logic to stats-module-utm, so that the index file serves as wrapper. In this case, the requests wouldn't be sent if the site is not eligible.

## Testing Instructions

* Open Calypso Live
* Ensure the UTM cards on Traffic page and single post page work for sites with valid subscriptions or internal sites
* Ensure it shows the upsell button for other sites and doesn't require a lot of loading time

## Pre-merge Checklist
